src/main/java/classify/user/InputParsing.java

```diff
@@ -517,4 +521,20 @@ private static boolean isDateValid(LocalDate paymentDate) {
 
     }
 
+    public static void checkForSpecialCharacters(String string)
+            throws InvalidCharacterException {
+        if (string == null) {
+            return;
+        }
+
+        if (string.contains("~")
```

Comment:
Good use of checking for banned characters to prevent users from interfering with the save file regex.
![image](https://github.com/AY2324S2-CS2113-T13-3/tp/assets/110646458/16eafc73-b5d3-40ba-9c56-d2961f796acd)